packages/flutter_tools/lib/src/test/flutter_web_platform.dart

```diff
@@ -668,7 +667,7 @@ class BrowserManager {
   /// Loads [_BrowserEnvironment].
   Future<_BrowserEnvironment> _loadBrowserEnvironment() async {
     return _BrowserEnvironment(
-        this, null, _browser.remoteDebuggerUri, _onRestartController.stream);
+        this, null, null, _onRestartController.stream);
```

Comment:
This might have been too breaking. I can updated to a separate lookup so that it does not affect the run workflow